app/src/main/java/ch/epfl/sdp/drone3d/ui/mission/ItineraryShowActivity.kt

```diff
@@ -76,6 +77,7 @@ class ItineraryShowActivity : BaseMapActivity() {
     private lateinit var area: List<LatLng>
     private lateinit var strategy: MappingMissionService.Strategy
     private var flightHeight: Double = 50.0
+    private var cameraPitch: Float = 0f
```

Comment:
Could cameraPitch and flightHeight be lateinit var instead of some default values ?

Reply:
I cannot put latent var on primitive values such as int and double